src/main/java/at/hannibal2/skyhanni/features/misc/CollectionTracker.kt

```diff
@@ -32,6 +35,8 @@ object CollectionTracker {
     private var display: Renderable? = null
 
     private var itemName = ""
+    private var isDungeonBoss = false
```

Comment:
instead of storing the boolean directly, rather use `val isDungBoss get() = dungeonFloor != null`